apps/backend/src/modules/ai-assistant/libs/helpers/initial-chat/generate-questions-answers-prompt.ts

```diff
@@ -0,0 +1,40 @@
+import { ZERO_INDEX } from "~/libs/constants/constants.js";
+import {
+	type OpenAiRequestMessage,
+	OpenAiRoleKey,
+} from "~/libs/modules/open-ai/open-ai.js";
+import { type OnboardingQuestionEntity } from "~/modules/onboarding/onboarding.js";
+
+import { OpenAiInitialPromptTemplates } from "./generate-init-promt-message.enum.js";
+
+function generateQuestionsAnswersPrompt(
+	userQuestionsWithAnswers: OnboardingQuestionEntity[],
+): OpenAiRequestMessage {
+	const questionsWithAnswers = userQuestionsWithAnswers.map(
+		(questionEntity) => {
+			const { answers, label: question } = questionEntity.toObject();
+			const answer = answers[ZERO_INDEX]?.label;
+
+			return { answer, question };
+		},
+	);
+
+	/* eslint-disable perfectionist/sort-objects */
```

Comment:
Can't disable eslint rules

Reply:
I need to create a prompt for a specific order.
And linter sort it.

I can use  template literals, but I'm not sure that it makes code cleaner:

```
`{
"context: OpenAiInitialPromptTemplates.INIT_CHAT_CONTENT",
"questionsWithAnswers": ${JSON.stringify(questionsWithAnswers)}
}`
```

I tried another version but I steel need a linter not to change the order.

Reply:
Try to find a workaround, maybe by creating a helper, which will generate the prompt. We can't disable rules